store/router.go

```diff
@@ -130,6 +133,17 @@ func (r *Router) Put(ctx context.Context, cm commitments.CommitmentMode, key, va
 	}
 
 	if err != nil {
+		log.Error("Failed to write to EigenDA backend", "err", err)
+		// write to EigenDA failed, which shouldn't happen if the backend is functioning properly
+		// use the payload as the key to avoid data loss
+		if r.cacheEnabled() || r.fallbackEnabled() {
+			redundantErr := r.handleRedundantWrites(ctx, value, value)
```

Comment:
add a comment to clarify why `value` is passed in here?
First `value` is used instead of EigenDA's `commitment` and it'll be keccak'ed as key in redundant writes.